### DISCUSSION TITLE
Request: Indexing hiero-ledger/hiero for Copilot Chat

### DISCUSSION THREAD

**Discussion by rblinton** [hiero-ledger/hiero#12] [General]
Please consider indexing the repository for Copilot Chat.

https://docs.github.com/en/copilot/managing-copilot/managing-github-copilot-in-your-organization/customizing-copilot-for-your-organization/indexing-repositories-for-copilot-chat#about-indexing-repositories

**Comment by hendrikebbers** [comment]
Good point. We will come back to it once all SDKs are transfered.

**Comment by jwagantall** [comment]
Looking into this, to trigger a re-index (which happens as a background task) we can do it by performing a search in the repository.
All we need to do is trigger a search which anyone can do.
